std/file.d

```diff
@@ -946,13 +947,14 @@ if (isConvertibleToString!RF || isConvertibleToString!RT)
     assert(t2.readText == "2");
 }
 
-private void renameImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, const(FSChar)* toz) @trusted
+private void renameImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, const(FSChar)* toz, Flag!"canCopy" canCopy) @trusted
```

Comment:
std/file.d(950:116)[warn]: Line is longer than 120 characters

Reply:
going to refactor this based on @jmdavis  comments (no need to review this code since it's all going to change)